src/assets/main.css

```diff
@@ -139,6 +139,7 @@ ul.btn-group {
 }
 .control.checkbox-container {
   flex-flow: unset;
+  gap: 0.15rem 1rem;
```

Comment:
This affects all checkboxes in the application. Did you make sure we want this accross the board?

Reply:
I did this because otherwise they cram together without spacing when placed side-by-side. It seems to look OK, what do you think?

Reply:
It's too wide for my taste to be applied across the app.
A more conservative gap would be a nice addition, though. Some of the labels elements have extra spaces in them which looks bad in template code.

Beyond the scope of this PR anyway. Please include the changes you need under `<style scoped>` in MidiView.vue.